content/developer/reference/frontend/framework_overview.rst

```diff
@@ -195,7 +195,7 @@ every 5 second:
 Components and Hooks
 --------------------
 
-Components and hooks are ideas coming from the
+:ref:`Components <frontend/components>` and :ref:`hooks <frontend/hooks>` are ideas coming from the
```

Comment:
FYI, there exists a `:doc:` command to reference a document instead of a hyperlink reference.

```:doc:`path_to_file` ```  -> ```:doc:`hooks` ```

I'm not saying you should use this instead of hyperlink references, because hyperlink refs are not broken by file moves, and are easier to use when referencing documents far from current one but it's good to know ;).

Reply:
good to know indeed. I actually kind of knew it, but didn't even think of it.  In this case, I think i'll keep the refs.